src/renderer/runner.ts

```diff
@@ -20,11 +20,16 @@ export enum ForgeCommands {
   MAKE = 'make',
 }
 
+interface runOptions {
+  runFromAsar: boolean;
+}
```

Comment:
```suggestion
export interface RunOptions {
  runFromAsar?: boolean;
}
```

Should start with a capital letter to match existing style, and should be exported since it's part of the public API for this file. Since these are options, `runFromAsar` should be optional as more options may be added in the future.

Reply:
Oops, missed the capital letter there, thanks for catching that! 
Exported it and made `runFromAsar` optional.